addons/account/models/account_move.py

```diff
@@ -1811,6 +1822,81 @@ def _get_creation_message(self):
             'in_receipt': _('Purchase Receipt Created'),
         }[self.move_type]
 
+    # -------------------------------------------------------------------------
+    # RECONCILIATION METHODS
+    # -------------------------------------------------------------------------
+
+    def _collect_tax_cash_basis_values_from_move(self):
+        ''' Collect all information needed to create the tax cash basis journal entries:
+        - Determine if a tax cash basis journal entry is needed.
+        - Compute the lines to be processed and the amounts needed to compute a percentage.
+        :return: A dictionary:
+            * move:                     The current account.move record passed as parameter.
+            * to_process_lines:         An account.move.line recordset being not exigible on the tax report.
+            * currency:                 The currency on which the percentage has been computed.
+            * total_balance:            sum(payment_term_lines.mapped('balance').
+            * total_residual:           sum(payment_term_lines.mapped('amount_residual').
+            * total_amount_currency:    sum(payment_term_lines.mapped('amount_currency').
+            * total_residual_currency:  sum(payment_term_lines.mapped('amount_residual_currency').
+            * is_fully_paid:            A flag indicating the current move is now fully paid.
+            * skip_check_rounding:      A flag indicating the code ensuring the full amount is exactly covered by the
+                                        tax cash basis journal entries must be skipped.
```

Comment:
that sentence is still not english, isn't it?

Reply:
If you say so :(

Reply:
You are right. I updated the docstring